tests/particle_interpolator_time_dependence.prm

```diff
@@ -0,0 +1,114 @@
+# This is a variation of the particle_interpolator_cell_average_2
+# test that includes a time varying solution with active particles.
+# In particular this test was introduced to check for the accuracy
+# of the interpolation in time-varying flow.
+
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1
+set Use years in output instead of seconds = false
+
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 2
+    set Y extent = 1
+  end
+end
+
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+  set List of model names = box
+
+  subsection Box
+    set Bottom temperature = 1
+    set Top temperature    = 0
+  end
+end
+
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left, right, bottom, top
+end
+
+
+subsection Gravity model
+  set Model name = vertical
+end
+
+
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Variable names      = x,z
+    set Function expression = (1-z)
+  end
+end
+
+
+subsection Material model
+  set Model name = simple
+
+  subsection Simple model
+    set Thermal conductivity                           = 1
+    set Thermal expansion coefficient                  = 0.01
+    set Viscosity                                      = 1
+    set Reference density                              = 1
+    set Reference temperature                          = 0
+    set Density differential for compositional field 1 = 100
+  end
+end
+
+subsection Discretization
+  set Use discontinuous composition discretization = true
+end
+
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 5
+  set Time steps between mesh refinement = 0
+end
+
+
+subsection Postprocess
+  set List of postprocessors = particles, velocity statistics, composition statistics, temperature statistics, visualization
+
+  subsection Visualization
+    set List of output variables = material properties
+    set Time between graphical output = 0
+  end
+
+  subsection Particles
+    set Number of particles        = 20000
+    set Time between data output = 0.03
+    set Data output format       = none
+    set List of particle properties = initial composition
+    set Update ghost particles   = true
+  end
+end
+
+
+# This is the new part: We declare that there will
+# be two compositional fields that will be
+# advected along. Their initial conditions are given by
+# a function that is one for the lowermost 0.2 height
+# units of the domain and zero otherwise in the first case,
+# and one in the top most 0.2 height units in the latter.
```

Comment:
Consider removing this comment, as the features below are not the "new part" of this test.